ipld/plugin/nmt.go

```diff
@@ -31,7 +31,7 @@ const (
 	Sha256Namespace8Flagged = 0x7701
 
 	// nmtHashSize is the size of a digest created by an NMT in bytes.
-	nmtHashSize = 2*consts.NamespaceSize + sha256.Size
+	nmtHashSize = 2*appconsts.NamespaceSize + sha256.Size
```

Comment:
[for reviewers] all the `consts` from celestia-core moved to `appconsts` in celestia-app